demo/site/src/layout/header/MobileMenu.tsx

```diff
@@ -0,0 +1,287 @@
+"use client";
+import { SvgUse } from "@src/common/helpers/SvgUse";
+import { PageLink } from "@src/layout/header/PageLink";
+import { PageLayout } from "@src/layout/PageLayout";
+import { useEscapeKeyPressed } from "@src/util/useEscapeKeyPressed";
+import { useState } from "react";
+import FocusLock from "react-focus-lock";
+import { FormattedMessage, useIntl } from "react-intl";
+import styled, { css } from "styled-components";
+
+import { type GQLMobileMenuFragment } from "./MobileMenu.fragment.generated";
+
+interface Props {
+    menu: GQLMobileMenuFragment;
+}
+
+export const MobileMenu = ({ menu }: Props) => {
+    const intl = useIntl();
+    const [isMenuOpen, setIsMenuOpen] = useState<boolean>(false);
+    const [expandedSubLevelNavigation, setExpandedSubLevelNavigation] = useState<string | null>(null);
+
+    const handleMenuButtonClick = () => {
+        if (isMenuOpen) {
+            setExpandedSubLevelNavigation(null);
+            setIsMenuOpen(false);
+        } else {
+            setIsMenuOpen(true);
+        }
+    };
+
+    const handleSubLevelNavigationButtonClick = (id: string) => {
+        if (expandedSubLevelNavigation === id) {
+            setExpandedSubLevelNavigation(null);
+        } else {
+            setExpandedSubLevelNavigation(id);
+        }
+    };
+
+    useEscapeKeyPressed(() => {
+        if (expandedSubLevelNavigation !== null) {
+            setExpandedSubLevelNavigation(null);
+        } else {
+            setIsMenuOpen(false);
+        }
+    });
+
+    return (
+        <Root>
+            <MenuButton
+                aria-label={intl.formatMessage({
+                    id: "header.menu.arialLabel",
+                    defaultMessage: "Menu",
+                })}
+                aria-expanded={isMenuOpen}
+                onClick={handleMenuButtonClick}
+            >
+                <SvgUse
+                    href={isMenuOpen ? "/assets/icons/menu-open.svg#root" : "/assets/icons/menu.svg#root"}
+                    width={16}
+                    height={16}
+                    color="inherit"
```

Comment:
Isn't color: inherit the default? You set it 6 times in this PR and it was net set before.